openreview/stages/arr_content.py

```diff
@@ -1495,6 +1495,17 @@
                 "input": "checkbox"
             }
         }        
+    },
+    "secondary_reviewer": {
```

Comment:
is the secondary reviewer always an openreview user?

Reply:
Yes it should be - the field was first added by the ARR tech team in one of the cycles and they used the `profile[]` type